### PR TITLE
add in-transaction for upgrade

### PR DESCRIPTION
**Pull request by evstratbg**
I needed to create index concurrently, but it was impossible because of the transaction. I fixed it

### CODE REVIEW

**Comment by evstratbg** [comment]
@long2ice hey man, really missing this feature. Any comments maybe?

File: aerich/cli.py [hunk 1]
```diff
@@ -90,11 +90,17 @@ async def migrate(ctx: Context, name):
 
 
 @cli.command(help="Upgrade to specified version.")
+@click.option(
+    "--in-transaction",
```
Comment (long2ice) [inline aerich/cli.py:94]: What about add `-i` also?

**Comment by long2ice** [comment]
Also please add the changelog

File: aerich/cli.py (hunk 1 above)
Comment (evstratbg) [reply aerich/cli.py:94]: added

**Comment by evstratbg** [comment]
@long2ice please, take a look one more time

**Comment by long2ice** [comment]
Need some format

**Comment by long2ice** [comment]
Thanks!

**Pull request merged**